src/app/components/common/dialog/jobs-manager/jobs-manager.store.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { EMPTY, Observable } from 'rxjs';
+import {
+  catchError, takeUntil, tap,
+} from 'rxjs/operators';
+import { JobsManagerState } from 'app/components/common/dialog/jobs-manager/interfaces/jobs-manager-state.interface';
+import { JobState } from 'app/enums/job-state.enum';
+import { Job } from 'app/interfaces/job.interface';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { DialogService, WebSocketService } from 'app/services';
+
+const initialState: JobsManagerState = {
+  isLoading: false,
+  jobs: null,
+  runningJobs: null,
+  failedJobs: null,
```

Comment:
Instead of storing runningJobs and failedJobs in state, it's better to make these a selector and calculate dynamically.
```
readonly numberOfRunningJobs$: Obserable<number> = this.select((state) => state.jobs.filter((job) => job.state === JobState.Running).length);
```